packages/angular/projects/ui-angular/src/lib/primitives-poc/phone-number/phone-number.module.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { AmplifyPhoneNumberFieldComponent } from './phone-number-component/phone-number.component';
+import { PhoneNumberDirective } from './phone-number-directive/phone-number.directive';
+
+let Components: any[] = [
```

Comment:
Do you know if we can avoid `any[]` here?

Reply:
We make it const and directly assign the array, Its constant so no need to give any[].

Reply:
[075752a](https://github.com/aws-amplify/amplify-ui/pull/2420/commits/075752a0a6eb028fab66163ccc05ff6c0e625e94)